conformance/tests/specialtypes_any.py

```diff
@@ -87,4 +88,99 @@ def method1(self) -> int:
 assert_type(a.method2(), Any)
 assert_type(ClassA.method3(), Any)
 
+# > When ``Any`` is present in the bases of a type,
+# > it should be considered only after all other known types in the MRO.
 
+class ClassKnown:
+
+    classvar1 = ""
+
+    def __iter__(self) -> Iterator[str]:
+        yield from self.attr1
+
+    def __init__(self):
+        self.attr1: str = ""
+
+    def method1(self) -> str:
+        return ""
+
+class AnyFirst(Any, ClassKnown):
+
+    def method2(self) -> str:
+        return ""
+
+class AnyLast(ClassKnown, Any):
+    def method2(self) -> str:
+        return ""
+
+class GetattrKnown(ClassKnown):
+    def __getattr__(self, name: str) -> int:
+        return 1
+
+class AnyFirstGetAttr(Any, GetattrKnown):
+    def method2(self) -> str:
+        return ""
+
+class AnyLastGetAttr(GetattrKnown, Any):
+    def method2(self) -> str:
+        return ""
+
+class AnySub(Any):
+    ...
+
+# primarily included to demonstrate intent that this is for the full MRO
+class AnySubFirst(AnySub, ClassKnown):
+    def method2(self):
+        ...
+
+
+
+af = AnyFirst()
+assert_type(af.method1(), str)
+assert_type(af.method2(), str)
+assert_type(af.attr1, str)
+assert_type(af.non_exist_method(), Any)
+assert_type(af.non_exist_attr, Any)
+assert_type(af.classvar1, str)
+assert_type(AnyFirst.classvar1, str)
+assert_type(iter(af()), Iterator[str])
```

Comment:
Just noticed this -- why are we calling `af()` here? Given that there's no explicit `__call__` defined above, it seems like perhaps this should not be a type error (since we assume `Any` is implicitly callable), but should be typed as `Any`?

It looks like this is intended to check the `__iter__` method above, in which case I think these parens should be eliminated
```suggestion
assert_type(iter(af), Iterator[str])
```

Reply:
This does raise the question of whether it's useful to also specify here the implicit `__call__` on classes with `Any` in MRO?

Reply:
I don't know if the implicit `__call__` should be documented or not, but the iter mistake was fixed.